assets/app/styles/_core.less

```diff
@@ -112,9 +122,18 @@ html, body {
       float: left;
       width: @navbar-project-menu-sm;
     }
+    .navbar-filter-widget {
+      /* prevent the filter button from wrapping at wide
+         screen sizes */
+      min-width: 340px;
+    }
     .navbar-search {
       margin-left: @sidebar-left-width-open-sm;
     }
+    .create-button-container {
+      text-align: right;
+      margin: 30px 0;
```

Comment:
when the 30px is there to align with some other peer element in the html, I'd almost rather keep it inlined... locality of reference and all